packages/lwc-language-server/src/metadata-utils/__tests__/content-assets-util.test.ts

```diff
@@ -1,12 +1,17 @@
-import { mockFileUtil } from './mock-file-util';
 import { indexContentAssets } from '../content-assets-util';
 import { validate } from './util';
-// @ts-ignore
-import { WorkspaceContext } from 'lightning-lsp-common';
+import { FORCE_APP_ROOT } from '../../__tests__/test-utils';
+import * as fs from 'fs-extra';
 
-jest.mock('lightning-lsp-common', () => {
-    const real = jest.requireActual('lightning-lsp-common');
-    return { utils: mockFileUtil(), WorkspaceContext: real.WorkspaceContext };
+beforeEach(() => {
+    const jsconfigPathForceApp = FORCE_APP_ROOT + '/lwc/jsconfig.json';
```

Comment:
if this has to happen before each unit test can we extract it to a shared function or something